vector/src/main/java/im/vector/app/core/di/SingletonModule.kt

```diff
@@ -184,4 +185,8 @@ object VectorStaticModule {
     fun providesAnalyticsConfig(): AnalyticsConfig {
         return analyticsConfig
     }
+
+    @Provides
+    @Singleton
+    fun providesBuildMeta() = BuildMeta()
```

Comment:
Introduces a build meta abstraction to allow for providing Build version overrides for unit tests